### PR TITLE
[CF-175] Create a cinder client with region

### PR DESCRIPTION
**Pull request by SVilgelm**

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests failed
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/2364/

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferrylib.os.storage.cinder_storage
cloudferrylib/os/storage/cinder_storage.py:140: [W0221(arguments-differ), CinderStorage.deploy] Arguments number differs from overridden 'deploy' method
cloudferrylib/os/storage/cinder_storage.py:319: [W0613(unused-argument), CinderStorage.convert_snapshot] Unused argument 'cloud'

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferrylib.os.storage.cinder_storage
cloudferrylib/os/storage/cinder_storage.py:140: [W0221(arguments-differ), CinderStorage.deploy] Arguments number differs from overridden 'deploy' method
cloudferrylib/os/storage/cinder_storage.py:319: [W0613(unused-argument), CinderStorage.convert_snapshot] Unused argument 'cloud'
************* Module tests.cloudferrylib.os.storage.test_cinder_storage
tests/cloudferrylib/os/storage/test_cinder_storage.py:241: [W0703(broad-except), CinderStorageTestCase.test_get_volume_path_iscsi_error] Catching too general exception Exception

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: cloudferrylib/os/storage/cinder_storage.py [hunk 1]
```diff
@@ -86,16 +86,17 @@ def cinder_client(self):
...
-        )
+        client_args = [params.cloud.user, params.cloud.password,
+                       params.cloud.tenant, params.cloud.auth_url]
+
+        client_kwargs = {"cacert": params.cloud.cacert,
+                         "insecure": params.cloud.insecure}
+        if params.cloud.region:
+            client_kwargs["region_name"] = params.cloud.region
+
+        return cinder_client.Client(*client_args, **client_kwargs)
```
Comment (antonf) [inline cloudferrylib/os/storage/cinder_storage.py]: Why not pass None if no region is specified instead of this complex code with kwargs?

Comment (SVilgelm) [inline cloudferrylib/os/storage/cinder_storage.py]: this is a quick fix, i just copied from nova client to check.

Comment (SVilgelm) [inline cloudferrylib/os/storage/cinder_storage.py]: Done

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferrylib.os.storage.cinder_storage
cloudferrylib/os/storage/cinder_storage.py:140: [W0221(arguments-differ), CinderStorage.deploy] Arguments number differs from overridden 'deploy' method
cloudferrylib/os/storage/cinder_storage.py:319: [W0613(unused-argument), CinderStorage.convert_snapshot] Unused argument 'cloud'
************* Module tests.cloudferrylib.os.storage.test_cinder_storage
tests/cloudferrylib/os/storage/test_cinder_storage.py:242: [W0703(broad-except), CinderStorageTestCase.test_get_volume_path_iscsi_error] Catching too general exception Exception

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by antonf** [comment]
+1

**Comment by roman-verchikov** [comment]
+1, can be merged

**Pull request merged**